sirepo/template/srw.py

```diff
@@ -456,6 +456,8 @@ def get_application_data(data, **kwargs):
 
 
 def get_data_file(run_dir, model, frame, **kwargs):
+    if frame < 0:
```

Comment:
This seems like an add test. Is there no other argument you could use in kwargs to make this be more explicit?

Reply:
What about passing in a string to indicate that it's a SRW log file: `'isSRWLogFile'`

Reply:
The call limits you at the moment to options.suffix. I think that could be a better signal than frame < 0. Look at [zgoubi.get_data_file](https://github.com/radiasoft/sirepo/blob/868fed4ad90e3c2503664528493447f89073162b/sirepo/template/zgoubi.py#L399) which abuses suffix to be the whole file name.